test/runtests.jl

```diff
@@ -368,27 +368,27 @@ end
     nullmod = lm(@formula(Result~1), d).model
     @test GLM.issubmodel(nullmod, mod)
     @test !GLM.issubmodel(othermod, mod)
-    
+
     @test_throws ArgumentError ftest(mod, othermod)
-    
+
     ft = ftest(mod, nullmod)
     @test isapprox(ft.pval[1].v,  2.481215056713184e-8)
     # Test output
-    @test sprint(show, ftest(mod, nullmod)) == 
+    @test sprint(show, ftest(mod, nullmod)) ==
         """
                 Res. DOF DOF ΔDOF    SSR    ΔSSR      R²    ΔR²       F* p(>F)
-        Model 1       10   3      0.1283          0.9603                      
```

Comment:
The trailing whitespace here will need to be preserved since it's inside of a string. Below (line 391) as well.